special-order-numbers.md

```diff
@@ -1,3 +1,11 @@
 # Create a Special Order for Telephone Numbers
 
-Details on how to place special orders for telephone numbers will be available soon.
+Customers who are interested in a special order (ex. acquiring large quantities of numbers) can do so by following the process below. 
+
+*PLEASE NOTE:* You need to have an eligible Azure Subscription and Resource in the US (see link here for eligibility criteria) to be able to purchase numbers. 
```

Comment:
We'll need to add the link here for link to eligibility criteria...   https://docs.microsoft.com/en-us/azure/communication-services/concepts/telephony-sms/plan-solution#azure-subscriptions-eligibility